contracts/PartyBuy.sol

```diff
@@ -25,8 +25,8 @@ contract PartyBuy is Party {
 
     // ============ Internal Constants ============
 
-    // PartyBuy version 1
-    uint16 public constant VERSION = 1;
+    // PartyBuy version 2
```

Comment:
Lets make sure to bring this back to version 1, and add the partyType() method (peep the discord thread)

Reply:
as well as renaming everything from PartyBuy to CollectionBuy or PartyCollectionBuy or something